hero/silk.json

```
@@ -0,0 +1,287 @@
+                        { "item": "gold", "qty": 33000 }
+                    ]
+                },
+                {
+                    "description": "+5% damage dealt by Automatic Fire",
+                    "resources": [
+                        { "item": "demon-blood-gem", "qty": 2 },
+                        { "item": "molagorago", "qty": 1 },
+                        { "item": "gold", "qty": 80000 }
+                    ]
+                }
+            ],
+            "buffs": [],
+            "debuffs": []
+        },
+        {
+            "isPassive": false,
+            "soulBurn": 0,
+            "selfSkillBarValue": 2,
+            "soulBurnEffect": "",
+            "awakenUpgrade": true,
+            "cooldown": 5,
+            "name": "Storm Arrow",
+            "soulAcquire": 2,
+            "description": "Attacks all enemies with wind, with a 80% chance to decrease Speed for 2 turns, and granting increased Speed to all allies for 2 turns.",
+            "enhancement": [
+                {
+                    "description": "+10% damage dealt",
+                    "resources": [{ "item": "molagora", "qty": 1 }, { "item": "gold", "qty": 4000 }]
+                },
+                {
+                    "description": "-1 turn cooldown",
+                    "resources": [
+                        { "item": "flame-of-soul", "qty": 1 },
+                        { "item": "molagora", "qty": 4 },
+                        { "item": "gold", "qty": 21000 }
+                    ]
+                },
+                {
+                    "description": "+15% effect chance",
+                    "resources": [
+                        { "item": "flame-of-soul", "qty": 1 },
+                        { "item": "molagora", "qty": 7 },
+                        { "item": "gold", "qty": 33000 }
+                    ]
+                },
+                {
+                    "description": "+15% damage dealt",
+                    "resources": [
+                        { "item": "demon-blood-gem", "qty": 2 },
+                        { "item": "molagorago", "qty": 1 },
+                        { "item": "gold", "qty": 80000 }
+                    ]
+                }
+            ],
+            "buffs": ["stic_speed_up"],
+            "debuffs": ["stic_speed_dn"]
+        }
+    ],
+    "specialtySkill": {
+        "name": "Wind Direction",
+        "description": "Maybe it sounds like wind, but it's really a song sent from the forest.",
+        "dispatch": ["[Explore] Type"],
+        "enhancement": ["Reward Bonus +6%"],
+        "stats": { "command": 20, "charm": 87, "politics": 40 }
+    },
+    "memoryImprint": [
+        {
+            "rank": "d",
+            "status": {
+                "type": "atk",
+                "increase": 0
+            }
+        },
+        {
+            "rank": "c",
+            "status": {
+                "type": "atk",
+                "increase": 0

```

Pretty late to notice, but do we have a way to tell the formation locations.
Like Silk's
....[..]
[..]...[X]
....[X]

```suggestion
                "increase": "2.9%"
```